dataplane/forwarding/fwdport/ports/group.go

```diff
@@ -56,7 +56,7 @@ func (m *member) String() string {
 
 // Write writes a packet out of the member after applying the member actions.
 func (m *member) Write(packet fwdpacket.Packet, caller string) error {
-	packet.Logf(fwdpacket.LogDesc, fmt.Sprintf("%v: %v %v:", m.ctx.ID, caller, m.port.ID()))
+	packet.Log().WithValues("context", m.ctx.ID, "caller", caller, "port", m.port.ID())
```

Comment:
WithValues only returns a logger. Call Info() to print the log?

Reply:
this is intentional, WithValues mutates the log sink so that subsequent logs have those values